src/Command/PqCommands/ContentHubPqSingleTranslation.php

```diff
@@ -0,0 +1,187 @@
+    $this->setAliases(['ach-pq-st']);
+    $this->addOption('entity-type', 'e', InputOption::VALUE_OPTIONAL, 'Run checks for the provided content entity type. Example node,user,paragraph');
+    $this->addUsage('ach:pq:non-translatables --entity-type "node,paragraph"');
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * @throws \Exception
+   */
+  protected function runCommand(InputInterface $input, PqCommandResult $result): int {
+    /** @var \Drupal\Core\Language\LanguageManagerInterface $languageManager */
+    $languageManager = $this->serviceFactory->getDrupalService('language_manager');
+    $languages = $languageManager->getLanguages();
+    if (count($languages) < 2) {
+      $result->setIndicator(
+        'Single translated entites',
+        '',
+        'No multiple languages detected.'
+      );
+      return 0;
+    }
+
+    $entityTypeOption = $input->getOption('entity-type');
+    /** @var \Drupal\Core\Entity\EntityTypeManagerInterface $etm */
+    $etm = $this->serviceFactory->getDrupalService('entity_type.manager');
+    /** @var \Drupal\Core\Entity\EntityTypeBundleInfoInterface $bundleInfo */
+    $bundleInfo = $this->serviceFactory->getDrupalService('entity_type.bundle.info');
+    $entityTypeDefinitions = $etm->getDefinitions();
+    $entityTypes = empty($entityTypeOption) ? array_keys($entityTypeDefinitions) : explode(',', $entityTypeOption);
+    foreach ($entityTypes as $entityType) {
+      // Skip config entity types.
+      if (array_key_exists($entityType, $entityTypeDefinitions) && $entityTypeDefinitions[$entityType]->entityClassImplements(ConfigEntityInterface::class)) {
+        continue;
+      }
+      $entityTypeLabel = $this->getEntityTypeLabel($entityType, $etm);
+      $bundles = $this->getBundles($entityType, $bundleInfo);
+      $kriName = 'Content Entity Type: ' . $entityTypeLabel;
+      if (empty($bundles)) {
+        $result->setIndicator(
+          $kriName,
+          '',
+          'No non-translatable entities detected.'
+        );
+        continue;
+      }
+
+      $entities = $this->loadEntities($etm, $entityType);
+      $singleTranslation = [];
+      foreach ($entities as $entity) {
+        if ($entity->isTranslatable() && count($entity->getTranslationLanguages()) === 1) {
+          $singleTranslation[$entity->bundle()]['count']++;
+          $singleTranslation[$entity->bundle()]['langcode'] = $entity->language()->getId();
+
+        }
+      }
+
+      $formatted = [];
+      foreach ($singleTranslation as $bundleId => $bundle) {
+        $formatString = $this->toYellow('%s: %s: %s');
+        $formatted[] = sprintf($formatString, $bundles[$bundleId]['label'], $bundle['langcode'], $bundle['count']);
+      }
+
+      $kriMessage = !empty($formatted) ? PqCommandResultViolations::$singleTranslation : 'No single-translation entities.';
+      $result->setIndicator(
+        $kriName,
+        implode("\n", $formatted),
+        $kriMessage,
+        !empty($formatted)
+      );
+    }
```

Comment:
I think we don't really have a violation here. What we want is just to collect the entity types and their respective bundles and print them. 
We can leave a message in the end that the customer should consider these as these languages for their respective bundles and entity types are going to be enabled.